src/main/java/app/ActionDispatcher.java

```diff
@@ -0,0 +1,7 @@
+package app;
+
+import server.request.Request;
+
+public interface ActionDispatcher {
```

Comment:
Even though it's not required, it can be useful to annotate your functional interfaces with `@FunctionalInterface` as it will tell the compiler to ensure it can be used as a functional interface.

Reply:
Thanks, that reminds me that I wanted to look into the purpose of the @ prefix in Java--I had only seen it in `@Before` and `@Test` until I saw it used otherwise in an example today.